integration-tests/src/main/java/io/druid/testing/clients/QueryResourceTestClient.java

```diff
@@ -93,6 +100,7 @@ private String getBrokerURL()
       );
     }
     catch (Exception e) {
+      log.error("exception: " + e.getMessage());
```

Comment:
@himanshug I agree with that assessment, but I also agree with @rasahner that merging logs when digging into errors is annoying. Would it appease you to have this log at a debug level?